Source/Scene/Model.js

```diff
@@ -186,6 +186,9 @@ define([
     function setCachedGltf(model, cachedGltf) {
```

Comment:
Just a heads up that we are going to have some merge conflicts when we merge this into the `3d-tiles` branch.